cfft/src/radix_2_butterfly.rs

```diff
@@ -0,0 +1,278 @@
+use alloc::vec::Vec;
+
+use p3_field::{ComplexExtension, Field};
+use p3_matrix::dense::RowMajorMatrix;
+use p3_matrix::Matrix;
+
+use crate::traits::CircleSubgroupFFT;
+use crate::util::{cfft_domain, twin_coset_domain};
+
+/// An O(N Log(N)) implementation of the CFT roughly analagous to the FFT approach to the DFT.
+/// See the paper "Circle Stark".
+#[derive(Default, Clone)]
+pub struct Radix2CFT;
```

Comment:
nit: should be `Radix2Cft` (as well for `CircleSubgroupFFT`)

Reply:
I've replaced `CircleSubgroupFFT` with `CircleSubgroupFt` as it seems more fitting. (It's a Fourier transform, not necessarily a fast Fourier transform one.